beetmoverscript/templates/firefox_candidates.yml

```diff
@@ -0,0 +1,129 @@
+---
+metadata:
+    name: "Beet Mover Manifest"
+    description: "Maps {{ product }} artifacts to pretty names for en-US"
+    owner: "release@mozilla.com"
+
+#TODO: we might s/firefox/{product} + some code logic tweak to get devedition for free
```

Comment:
Good idea. We don't have to do that now, but if devedition turns out to be close enough we can do that then.